org.etherpad.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id": "org.etherpad.etherpad-lite",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "etherpad-lite",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--share=ipc"
+  ],
+  "modules": [
+    {
+      "name": "etherpad-lite",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/ether/etherpad-lite/archive/refs/tags/v1.9.4.tar.gz",
+          "dest": "etherpad-lite",
+          "sha256": "e198214acdbb57363561dee2d73f27199999af26c283723067525bd854703e12",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1,
```

Comment:
This does not belong to etherpad. Use git checker or json checker.

Reply:
well it looks like this was cut and paste from an example given the template url.

Reply:
Yes. That is why it is a draft pull request. @Heus-Sueh asked if I could make etherpad available on FlatHub

Reply:
draft pull request are mean to signal "this work but still waiting on some stuff". Not mean to be for throwing thing on the wall and see if it sticks. One of the item in the checklist ask you to have built this locally.